### ISSUE TITLE
support markdown in Product Description in MarketPlace

### ISSUE THREAD

**Issue by Trophime** [MSO4SC/MSOPortal#85]
Would it be possible to support markdow in description fields (or eventually other like licenses...)
for Product and Offering?

**Comment by emepetres** [comment]
Do you mean in the Marketplace right?

**Comment by Trophime** [comment]
yep in part where we fill fields like Description, License and so on..

**Comment by Trophime** [comment]
![marketplace_offering_detail](https://user-images.githubusercontent.com/1309311/41142654-a9f72606-6af6-11e8-85ff-627ff5b82e4d.png)

note that it also happens in ckan:
![ckan_dataset_list](https://user-images.githubusercontent.com/1309311/41142673-bed45c60-6af6-11e8-8c57-209682858ca3.png)
